projects/aca-content/src/lib/components/info-drawer/comments-tab/comments-tab.component.ts

```diff
@@ -22,28 +22,28 @@
  * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import { MinimalNodeEntryEntity } from '@alfresco/js-api';
 import { NodePermissionService, isLocked } from '@alfresco/aca-shared';
 
 @Component({
   selector: 'app-comments-tab',
   template: `<mat-card><adf-node-comments [readOnly]="!canUpdateNode" [nodeId]="node?.id"></adf-node-comments></mat-card>`
 })
-export class CommentsTabComponent {
+export class CommentsTabComponent implements OnInit {
   @Input()
   node: MinimalNodeEntryEntity;
 
+  canUpdateNode = false;
+
   constructor(private permission: NodePermissionService) {}
 
-  get canUpdateNode(): boolean {
+  ngOnInit(): void {
     if (!this.node) {
```

Comment:
General comment for all places - when you use ngOnInit and results depends on Input fields (like here node) then please check if there is possibility that some action in parent changes Input value because then ngOnInit is not called and you may need recalculate results.

Reply:
I did keep that in mind and implemented re-evaluation of values via ngOnChanges in places where I saw the @Input fields getting changed after creation. However, you raise a good point and I'll double check if anything was missed.

Reply:
Thank you. If nothing is changing then keep it as it is. Just wanted to raise that to ensure we handle this kind of situations